modules/mauth-authenticator-apachehttp/README.md

```diff
@@ -15,6 +15,7 @@ This is an implementation of Medidata Authentication Client Authenticator to val
                 
         mauth {
             base_url: "http://localhost"
+            disable_v1: false
```

Comment:
as this is in `reference.conf`, it doesn't have to be provided, unless default value is overridden

Reply:
fixed https://github.com/mdsol/mauth-jvm-clients/pull/89/commits/639fbc9ed0fbf79345c5ddad83d1a8b379b68852